src/app/shared/functions/teamSheet.ts

```diff
@@ -1,3 +1,15 @@
+
+export const fantasyBench = [
```

Comment:
sent a message on slack regarding this, but could we just use this fantasy bench for regular teams?